core/src/main/java/io/questdb/std/Files.java

```diff
@@ -110,11 +93,12 @@ public static long ceilPageSize(long size) {
         return ((size + PAGE_SIZE - 1) / PAGE_SIZE) * PAGE_SIZE;
     }
 
-    public static int close(int fd) {
+    public static int close(long fd) {
         // do not close `stdin` and `stdout`
-        if (fd > 1) {
-            assert auditClose(fd);
-            int res = close0(fd);
+        int osFd;
+        if (fd != 0 && (osFd = toOsFd(fd)) > 1) {
```

Comment:
For completeness, why don't we handle `stderr` equally, FD = 2?

A related nit is that we assume stdout FD to be 1 here, but elsewhere use a native method `getStdOutFd` to resolve it.

Reply:
FD 2 will also be handled, changed in 5f22804767ebade94ec0efa41bd2ce172cc78160
I don't want to address the second comment in this PR. Assuming stdout FD == 1  has been working ok so far, the only exclusion is when the DB runs on Windows as non-console app, and it's possible in embedded scenario but let's address it when there will be real issue with it.